### PR TITLE
Cleanup verification-metadata

### PR DESCRIPTION
**Pull request by paul-dingemans**
## Description

Remove all components from the verification-metadata.xml which are no…t required for the build. As mentioned in #1313 some (reference to) log4j-core dependencies were found in the ktlint binary. Those dependencies were not actually used. It seems that the original verification-metadata.xml already contained a lot of trusted component which are not actually used.

Please note that I have little experience with gradle. But all build taks on the local machine are still working.

#Closes 1313

## Checklist

- [ ] tests are added
- [X] `CHANGELOG.md` is updated

### CODE REVIEW

**Comment by shashachu** [comment]
I'm going to leave this for @Tapchicoma to review; I also have little experience with this aspect of Gradle.

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 - Do not remove trailing comma in annotation ([#1297](https://github.com/pinterest/ktlint/issues/1297))
 - Do not remove import which is used as markdown link in KDoc only (`no-unused-imports`) ([#1282](https://github.com/pinterest/ktlint/issues/1282))
 - Fix indentation of secondary constructor (`indent`) ([#1222](https://github.com/pinterest/ktlint/issues/1222))
+- Cleanup trusted dependencies ([#1313](https://github.com/pinterest/ktlint/issues/1313))
```
Comment (romtsn) [inline CHANGELOG.md:18]:
```suggestion
```
Since it's a chore, let's leave it out from the changelog.

**Comment by Tapchicoma** [comment]
BTW interesting that `:ktlint` task has failed :thinking:

**Comment by paul-dingemans** [comment]
> BTW interesting that `:ktlint` task has failed 🤔

Yes, I noticed that as well on another branch today. A while ago we change the build process so that the ktlint task also runs all experimental rules on ktlint itself (dogfeeding principle). Somehow, the code on master at that moment contained some violations of the trailing comma rule which probably were introduced between submitting the PR and actually merging it into master. Anyway, I have fixed the violations in master in meanwhile.

**Pull request merged**